src/modules/menu/components/advancedTools/ImportFile/ImportFileOnlineTab.vue

```diff
@@ -0,0 +1,112 @@
+<script setup>
+import { computed, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import axios, { AxiosError } from 'axios'
+
+import { isValidUrl } from '@/utils/utils'
+import log from '@/utils/logging'
+import { handleFileContent } from '@/modules/menu/components/advancedTools/ImportFile/utils'
+import ImportFileButtons from '@/modules/menu/components/advancedTools/ImportFile/ImportFileButtons.vue'
+
+const BTN_RESET_TIMEOUT = 3000 // milliseconds
```

Comment:
as this logic is also present in the other type of input, maybe it would be a good idea to use some composable to isolate it and describe it only once (or move this timeout management straight into the `ImportFileButtons` component)